### DISCUSSION TITLE
linux-surface vs. 5.13 mainline kernel

### DISCUSSION THREAD

**Discussion by mbrennwa** [linux-surface/linux-surface#477] [General]
Im am writing this using my Surface Laptop 4 running on the release candidate of the Linux mainline kernel version 5.13. As far as I can tell, it works pretty well. However, I am wondering if the 5.13 mainline kernel will have everything related to the Surface which is in linux-surface. Am I missing anything by running 5.13 mainline kernel instead of the linux-surface one?

**Comment by qzed** [comment]
The Intel version of the SL4 (as well as SL3 and SB3) has all the necessary bits upstream, so, at the moment at least, you shouldn't be missing anything. The AMD variant of the SL4 still relies on some (not really Surface-specific) fixes that, unfortunately, aren't upstream (and it has some problems that still need to be ironed out).
